### PR TITLE
Task 11 completed

### PR DESCRIPTION
**Pull request by Szpytma**
- updated readme.md
- implemented queries for reviews this endpoint accepts 'category', 'sort_by' & 'order' as quarries.
Waiting for approval

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -88,6 +88,69 @@ describe("GET /api/reviews", () => {
...
+  });
+
+  it("400: ", () => {
+    return request(app)
+      .get("/api/reviews?category=not_a_category")
+      .expect(400)
+      .then(({ body }) => {
+        const { message } = body;
+        expect(message).toBe("Category not Found");
+      });
```
Comment (robert112233) [inline __tests__/app.test.js:222]: If the category is not found it should be 404 - not found

File: __tests__/app.test.js [hunk 2]
```diff
@@ -88,6 +88,69 @@ describe("GET /api/reviews", () => {
...
+  });
```
Comment (robert112233) [inline __tests__/app.test.js:223]: This is a good start on the tests. You are missing tests for:
400 invalid sortBy - sort_by=notTitle. 
400 invalid order - order=ascending&desc_at_the_same_time 

There is one happy path test 200: where you sort by a category that exists but has no associated reviews - returns []

**Review by robert112233 — CHANGES_REQUESTED** [review]
Good attempt. There are some more tests to include, make the changes and submit another PR for review.

File: __tests__/app.test.js (hunk 2 above)
Comment (Szpytma) [reply __tests__/app.test.js:223]: the last case took ages to fix! but its sorted now

File: README.md [hunk 3]
```diff
@@ -6,25 +6,19 @@ This will be an API for the purpose of accessing application data programmatical
 
 The database will be PSQL, and it will be possible to interact with it using [node-postgres](https://node-postgres.com/).
 
-To see that project please
+Link to the hosted version [Szpytma nc-games](https://nc-games-rkbx.onrender.com).
 
-1. run npm i / npm install to initialize all needed dependencies
-2. create .env files for the dev and test data in the root of the project folder (env.development, .env.test).
+## To see possible endpoints please go to api/
```
Comment (alexdevdavis) [inline README.md]: should this be `/api`

**Review by alexdevdavis — APPROVED** [review]
Looking good. 
- I'd suggest adding some mention of the node and postgres versions, for compatibility's sake

**Pull request merged**